stores/maker/selectors.js

```diff
@@ -0,0 +1,23 @@
+const updateEntity = (state, id, newProps) => ({
+  ...state,
+  appEntities: {
+    ...state.appEntities,
+    [id]: {
+      ...state.appEntities[id],
+      ...newProps
+    }
+  }
+})
+
+export const selectMaker = state => state.maker
+
+export const selectAppList = state => state.maker.appIds.map(id => state.maker.appEntities[id])
+
+export const selectCurrentApp = state => state.maker.appEntities[state.maker.selectedAppId]
+
+// export const selectCompetionStatus = state => {
+//   const selectedApp = state.appEntities[state.selectedAppId]
+//   return {
+//     paymentDetailsComplete
+//   }
+// }
```

Comment:
If we don't need this we should remove it before merging

Reply:
Sorry for the commented out stuff, I had some extra code to push that's stashed